src/Authoring/WinRT.SourceGenerator/WinRTTypeWriter.cs

```diff
@@ -1283,10 +1283,12 @@ private void EncodeTypedConstant(TypedConstant constant, LiteralEncoder encoder)
                     encoder.Scalar().Constant(constant.Value);
                     break;
                 case TypedConstantKind.Enum:
-                    encoder.TaggedScalar(
-                        type => type.Enum(constant.Type.ToString()),
-                        scalar => scalar.Constant(constant.Value)
-                    );
+                    encoder.Scalar().Constant(constant.Value);
+
+                    //encoder.TaggedScalar(
+                    //    type => type.Enum(constant.Type.ToString()),
+                    //    scalar => scalar.Constant(constant.Value)
+                    //);

```

Comment:
Let's include a comment for context, also let's leave a space after a comment.
```suggestion
                    // This looks more correct, but the code above matches what the C# compiler produces.
                    // Eg. win32metadata does the same (see 'MetadataUtils.EncodeHelpers' in https://github.com/microsoft/win32metadata).
                    // encoder.TaggedScalar(
                    //     type => type.Enum(constant.Type.ToString()),
                    //     scalar => scalar.Constant(constant.Value)
                    // );

```